### ISSUE TITLE
Is 'molecule-plugins[docker]' replacing 'molecule[docker]'?

### ISSUE THREAD

**Issue by LorenzoBettini** [ansible-community/molecule-plugins#103]
The README says something about `molecule-azure` that should be replaced by `molecule-plugins[azure]`. Now I seem to understand that this repository also includes the docker plugin. Does that mean that instead of `molecule[docker]` we should install `molecule-plugins[docker]`?

Thanks in advance

**Comment by apatard** [comment]
The azure part of the README is only an example to show that the molecule-plugin driver dependencies are not installed by default. So, to install molecule, the docker driver and its dependencies:
```
pip3 install molecule molecule-plugins molecule-plugins[docker]
```

**Comment by LorenzoBettini** [comment]
@apatard 

Isn't enough to install `molecule-plugins[docker]`? This should imply `molecule-plugins`, shouldn't it?

Moreover, `molecule[docker]` should be uninstalled first, shouldn't it?

**Comment by apatard** [comment]
yes, installing ``molecule-plugins[docker]`` should be enough. Answered too quickly.

I've not mentioned removing ``molecule-docker`` because if you're installing from scratch, it's not needed. In case of existing setup you're right, ``molecule-docker`` has to be removed.

**Comment by caripepo** [comment]
@LorenzoBettini were you able to successfully enable molecule with docker... if so what is your current setup??

python version
ansible version
molecule-plugin[docker] version
OS ?

I am having kinda of a similar concern was able to enable ansible, molecule and docker but with python3.9 but molecule login was failing and destroy the config.. Now I am trying to have the same setup but with python3.11 but every time I install molecule-plugin[docker], molecule breaks and so ansible.

**Comment by apatard** [comment]
@LorenzoBettini do you think this bug can be closed ?
@caripepo if you still have troubles, I think it would be better to open a new bug report with more details of what's failing for you.

**Comment by LorenzoBettini** [comment]
@apatard to be honest I still haven't got a chance to try the new `molecule-plugins[docker]`: I'm still using `molecule[docker]`.
